openlibrary/templates/books/check.html

```diff
@@ -53,7 +53,7 @@ <h3 class="booktitle"><a href="javascript:;" title="$_('Select this book')">$wor
 </div>
 
 <script type="text/javascript">
-<!--
+
```

Comment:
Was this code commented out for a reason?  Was the commenting removed for a reason?

Reply:
It is not the comment in the script the comment is js is written like 
```/* */``` not ```<--``` 
It was giving an error thats why I remove this line 
This type of comment is used in HTML not in script
![Screenshot (29)](https://user-images.githubusercontent.com/43555219/97802689-37e89580-1c6b-11eb-9efb-4ab156045d5a.png)